pkg/compactor/compactor.go

```diff
@@ -60,6 +60,17 @@ var (
 	errInvalidSymbolFlushersConcurrency           = fmt.Errorf("invalid symbols-flushers-concurrency value, must be positive")
 	errInvalidMaxBlockUploadValidationConcurrency = fmt.Errorf("invalid max-block-upload-validation-concurrency value, can't be negative")
 	RingOp                                        = ring.NewOp([]ring.InstanceState{ring.ACTIVE}, nil)
+
+	// compactionIgnoredLabels defines the external labels the compactor
+	// will drop/ignore when planning jobs.
+	// * We remove ingester ID because we don't shard blocks anymore, while still
+	// 	honoring the shard ID if sharding was done in the past.
```

Comment:
I know this is just copied from previous location, but honestly I don't really understand or remember what it's trying to say. Let's say that in the past Cortex used to add ingester ID label to the blocks, but it no longer does that, and we ignore the label just in case someone tries to use some very old Cortex-generated blocks with Mimir.

Reply:
Cool - really those justifications are more or less redundant with info already on those `DeprecatedXyz` constants, so I've just 🔪'd them.